src/Program.cs

```diff
@@ -8,17 +8,21 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Threading;
 
 namespace i3dm.export
 {
     class Program
     {
         static void Main(string[] args)
         {
+            Thread.CurrentThread.CurrentCulture = System.Globalization.CultureInfo.InvariantCulture;
```

Comment:
can be removed? use InvariantCulture when needed

Reply:
I'm not sure what the best option is. The coordinates in queries https://github.com/Geodan/i3dm.export/blob/feature/cesium-support/src/InstancesRepository.cs#L15 and https://github.com/Geodan/i3dm.export/blob/feature/cesium-support/src/InstancesRepository.cs#L37 result in wrong numbers on my machine, this is fixed by setting the thread to InvariantCulture.

I could add a function ToInvariantCulture and run all the coordinates trough this?